android/app/src/main/python/mmrelay/e2ee_utils.py

```
@@ -0,0 +1,392 @@
+        "overall_status": "unknown",
+        "issues": [],
+    }
+
+    # Check platform support
+    if sys.platform == WINDOWS_PLATFORM or sys.platform.startswith(("msys", "cygwin")):
+        status["platform_supported"] = False
+        status["issues"].append("E2EE is not supported on Windows")
+
+    # Check dependencies
+    try:
+        import olm  # noqa: F401
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)